README.md

```diff
@@ -244,39 +244,44 @@ You can also add them to the end of `~/.profile` to set this by default in all s
 
 #### MacOS dependencies installation
 
-To install GCC and CMake on MacOS we recommend to use [Homebrew](https://brew.sh/) package manager. With Homebrew
+Install Xcode Command Line Tools if you don't have them. Run:
+```sh
+xcode-select --install
+```
+Follow the prompts to contunue.
+
+To install GCC and CMake on macOS we recommend to use [Homebrew](https://brew.sh/) package manager. With Homebrew
 installed, run the following commands:
 ```sh
-brew install gcc cmake 
+brew install gcc@14 cmake 
```

Comment:
remove trailing spaces (they are also on the lines 263, 280, 282, 283)

Reply:
Seems, like this line wasn't updated

Reply:
Fixed